diff --git a/node_modules/expo-dev-menu/android/build.gradle b/node_modules/expo-dev-menu/android/build.gradle
index d108543..75c5ee5 100644
--- a/node_modules/expo-dev-menu/android/build.gradle
+++ b/node_modules/expo-dev-menu/android/build.gradle
@@ -182,7 +182,7 @@ dependencies {
   // Needed by gesture handler
   implementation "androidx.core:core-ktx:1.6.0"
 
-  api "androidx.browser:browser:1.6.0"
+  api "androidx.browser:browser:1.5.0"
 
   testImplementation "com.google.truth:truth:1.1.2"
   testImplementation "org.robolectric:robolectric:4.10"
